src/main/java/org/prebid/server/auction/BidResponseCreator.java

```diff
@@ -512,6 +522,57 @@ private Bid toBid(BidderBid bidderBid, String bidder, ExtRequestTargeting target
         return bid;
     }
 
+    private static void addNativeMarkup(Bid bid, List<Imp> imps) {
+        final Response nativeMarkup;
+        try {
+            nativeMarkup = Json.decodeValue(bid.getAdm(), Response.class);
+        } catch (DecodeException e) {
+            throw new PreBidException(e.getMessage());
```

Comment:
Isn't it too harsh? Seemingly there is no need to fail whole PBS response because of one malformed bid markup.

Reply:
Haven't thought about that... I guess I would have to change the flow a little bit so that any errors occurring here could be added to `response.ext`, while just skipping "problematic" bid.

Reply:
Yeah, this seems totally reasonable